### ISSUE TITLE
NTR, pulmo: polyalveolar lobe

### ISSUE THREAD

**Issue by GrieseAG** [obophenotype/human-phenotype-ontology#6435]
**Preferred term label:**
polyalveolar lobe

**Synonyms**
Increased number of alveoli

**Definition (free text, please give PubMed ID)**
In polyalveolar lobe, the total alveolar number as determined by microscopic point-counting of randomly taken lung sections is increased three to fivefold. The airways and arteries are normal for age in number, size and structure. 

Comment: All patients have some pulmonary function abnormalities, specifically a decrease in airway conductance, and forced expiratory volume.
PMID: 7463297

**Parent term (use hpo.jax.org/app)**
Abnormal pulmonary alveolar system morphology HP:0033137

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ORPHA:1928   (Disorder)   Congenital lobar emphysema

**Your nano-attribution (ORCID)**
lmu: chrapp, @kknoflach, @mgriese

**Comment by pnrobinson** [comment]
I have slightly reworded the definition

An elevation of the total count of alveoli as determined by microscopic point-counting of randomly taken lung sections, not secondary to abnormalities in number, size and structure of the airways. In polyaveolar lobe, the number of alveoli is generally increased three to fivefold.

please reopen this issue if the revised definition is incorrect. @chrapp, @kknoflach, @mgriese
